halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/Node.java

```diff
@@ -176,14 +177,27 @@ public void parentify() {
 
     while (clazz != null) {
       res.addAll(Arrays.stream(clazz.getDeclaredFields())
-          .filter(f -> f.getDeclaredAnnotation(LocalFile.class) != null)
+              .filter(f -> f.getDeclaredAnnotation(LocalFile.class) != null && !isSecretFile(f))
```

Comment:
If someone doesn't want to use the secret mgmt, does this break anything? We're now omitting a number of files that used to be included.

Reply:
It shouldn't. `isSecretFile` returns true only if the field is annotated with @SecretFile AND the content is a secret (or looks like one, `encrypted!...`)